modules/installation-azure-confidential-vms.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-confidential-vms_{context}"]
+= Enabling Confidential VMs
+
+:FeatureName: Confidential VM
+
+You can enable Confidential VMs for compute nodes, control plane nodes, or all nodes when installing your cluster. 
```

Comment:
Not critical, but I wonder if this would read better as two sentences? Something like saying that you can enable confidential VMs when installing your cluster. And then go into saying that you can enable it for all nodes, or just for compute notes, or just for control plane nodes